src/index.js

```diff
@@ -227,7 +234,15 @@ class WarmUP {
         : this.warmup.default
 
       /** Function needs to be warm */
-      return enable(functionConfig)
+      if (enable(functionConfig)) {
+        const concurrency = functionObject.hasOwnProperty('warmupConcurrency') && typeof functionObject.warmupConcurrency === 'number'
```

Comment:
I tested and confirmed that trying to use `functions.foobar.warmup.concurrency` doesn't work because `functions.foobar.warmup` is being used as a value.  For example, if you try this, serverless errors out:

```
functions:
  foobar:
    warmup: true
      concurrency: 2
```

So for now `function.foobar.warmupConcurrency` is used to avoid breaking compatibility and making changes beyond the depth of this PR.